### DISCUSSION TITLE
Is it work with Thinkpad T480s ?

### DISCUSSION THREAD

**Discussion by NexiaMoe** [valnoxy/t480-oc#45] [Q&A]
Hello, Thank you for your works, it's really awesome !

I have question about my laptop, Is it will work with T480s ? 

Specs is :
Prosesor : Intel Core i5-8250U (8th Gen)
Storage : SSD 256GB
Ram : 16GB
Screen : 14" (35.56 cm) display, 1920 x 1080 px
Grapich : Intel UHD 620

**Comment by jodave911** [comment]
👍🏼, will do.

**Reply by ramdani1978** [reply]
Mine works, but have shutdown problem, it always restart.

**Reply by jodave911** [reply]
Do check with this repo, 
https://github.com/askwakhid/ThinkPad-T480s-macOS-OpenCore

**Comment by valnoxy** [comment]
From the hardware side it should work, but some problems like RTC write issues may occur.
